packages/ckeditor5-remove-format/tests/manual/removeformat.md

```diff
@@ -1,10 +1,12 @@
 # Remove Format
 
+Select the image and resize it.
```

Comment:
It renders like that:

![](https://user-images.githubusercontent.com/5353898/121360629-e016b900-c934-11eb-934e-87d9024393bd.png)

I'd prefer it to be a list of steps to be more readable, like that:

> 1.  Select the image and resize it.
> 2.  Select the content and press the "Remove Format" button in the toolbar.

BUT, I think we can even make it simpler 🙂 Wouldn't it be better for testers if instead of having to resize the image every single time they would already have it resized?

All you need to do is to edit `removeformat.html` file and add `width` style and `image_resized` class to the figure.

```css
<figure class="image image_resized" style="width:18%;"><img src="assets/sample.png" alt="Autumn fields"><figcaption>Autumn fields by Aleksander Nowodziński</figcaption></figure>
```

With such change the image is already resized when the manual test is open 👌 Less steps = more time for other things.